app/Http/Controllers/Admin/UserDisableController.php

```diff
@@ -27,13 +27,13 @@ public function __construct(
         UserRepository $userRepository,
         ProgramRepository $programRepository,
         PositionRepository $positionRepository,
-        WorkspaceRepository $workspaceRepository)
-    {
+        WorkspaceRepository $workspaceRepository
+    ) {
         $this->userRepository = $userRepository;
         $this->programRepository = $programRepository;
         $this->positionRepository = $positionRepository;
         $this->workspaceRepository = $workspaceRepository;
-        $this->middleware('checkTrainer')->except('index', 'create', 'store' , 'edit' , 'update');
+        $this->middleware('checkTrainer')->except('index', 'create', 'store', 'edit' , 'update');
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.Functions.FunctionCallArgumentSpacing.SpaceBeforeComma</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/UserDisableController.php</br>Space found before comma in function call